java/pom.xml

```diff
@@ -118,6 +127,11 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>${commons-lang3.version}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-io</groupId>
+                <artifactId>commons-io</artifactId>
+                <version>${commons-io.version}</version>
+            </dependency>
```

Comment:
Can we update the NOTICES file with this new dependency?

Reply:
It's already there.